src/settings.cpp

```diff
@@ -829,9 +829,9 @@ directoryAddress = GetIniSetting ( IniXMLDocument, "server", "centralservaddr",
 
     // directory type
     // CServerListManager defaults to AT_NONE
-    // Server GUI defaults to AT_DEFAULT
+    // Server GUI also defaults to AT_NONE (previously AT_DEFAULT, when registration was controlled by a checkbox)
     // Because type could be AT_CUSTOM, it has to be set after the address to avoid multiple registrations
-    EDirectoryType directoryType = AT_DEFAULT;
+    EDirectoryType directoryType = AT_NONE;
 
     // if command line is set, use it
```

Comment:
> if command line is set, use it

Mm, I use too few words...

> If a command line Directory server address is set, set the Type to AT_CUSTOM so it's used

might work better...

Reply:
Sure, I'll add that.